src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackAssessment/ActivityCardItemList/ActivityCardItem/ActivityCardItem.types.ts

```diff
@@ -1,10 +1,10 @@
-import { ActivityItemAnswer } from 'shared/types';
+import { AssessmentActivityItem } from 'modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.types';
```

Comment:
could you pls import from index to decrease the length of path
also in other places